### ISSUE TITLE
Label edit is not shown until refresh

### ISSUE THREAD

**Issue by BasThomas** [rnystrom/Freetime#236]
Version 1.5.0 (1330)
Device: iPhone 7 Plus (iOS 10.3.3)

**Comment by rnystrom** [comment]
Can you give a little more detail? What isn't appearing exactly?

**Comment by BasThomas** [comment]
1. Edit a label
2. Expect an event "added" or "removed" for the label to be added (without refreshing the issue)
3. Actually need to refresh the issue to see label events

**Comment by Sherlouk** [comment]
Can confirm, having same experience

**Comment by rnystrom** [comment]
Ah ya, I left this out on purpose b/c of how much it will complicate timeline models. Not super high pri since the actual state changes.

If someone wants to take a stab at it, happy to review!

**Comment by weyert** [comment]
Maybe just automatically refresh the issue after editing a label?

**Comment by rnystrom** [comment]
True, I'm always a stickler for being minimal, but that might end up being the simplest.
